.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+default_language_version:
+    python: python3.9
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-added-large-files
+  - repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+    -   id: black
+  - repo: https://github.com/pycqa/flake8
+    rev: '7.1.1'
+    hooks:
+    - id: flake8
+# YAML stuff from https://earth.bsc.es/gitlab/digital-twins/de_340-2/workflow
+#  - repo: https://github.com/adrienverge/yamllint.git
+#    rev: v1.35.1 # or higher tag
+#    hooks:
+#      - id: yamllint
+#        args: [--format, parsable]
+#  - repo: https://github.com/google/yamlfmt
+#    rev: v0.10.0
+#    hooks:
+#      - id: yamlfmt
```

Comment:
We don't have much yaml in our project (yet), but having an auto-yaml linter might be very useful. I think I have something else for yaml at the moment -- (`yamlfmt`)[https://github.com/google/yamlfmt]. I suppose it is simple enough to swap out which program it runs?

Reply:
Yes, it is very easy, just let me know what would you prefer and I will include it :)

Reply:
Let's stick with what you already have.